common/archiver/util.go

```diff
@@ -30,7 +30,7 @@ import (
 
 var (
 	errEmptyDomainID         = errors.New("DomainID is empty")
-	errEmptyDomainName       = errors.New("Domain name is empty")
+	errEmptyDomainName       = errors.New("DomainName is empty")
```

Comment:
staticcheck complained about this?  or did it just complain about uppercase -> now it matches the param name, and staticcheck is happy because it looks like a ProperNoun or something?

Reply:
`staticcheck` complained about upper case. Since other errors here include exact param names, I changed this for consistency as well.

Out of curiosity checked their source code: https://github.com/dominikh/go-tools/blob/f41ebe766a412094c088d7d32598c19442db66c1/stylecheck/lint.go#L418

Seems like if it detects more upper letters in the first word - it is ok:
```
// Word is probably an initialism or
// multi-word function name
```